server/config/options.go

```diff
@@ -505,6 +508,8 @@ type SchemaConfig struct {
 
 // FoundationDBConfig keeps FoundationDB configuration parameters.
 type FoundationDBConfig struct {
+	// Chunking allows us to persist bigger payload in storage.
+	Chunking    bool   `mapstructure:"chunking" yaml:"chunking" json:"chunking"`
```

Comment:
I was thinking to add special KVConfig.

Reply:
Where we can put chunking, compression, stats, which is actually independent from FDB.

Reply:
Makes sense, added with chunking.